src/translations/translations_NO.json

```diff
@@ -50,6 +50,7 @@
     "or": "Eller",
     "other_options": "Andre lisenser",
     "page": "Page",
+    "preview_not_implemented": "Forhåndsvisning er ikke implementert",
```

Comment:
"Forhåndsvisning kommer snart" er kanskje bedre? Sånn i tilfellet testerne blir nå ville over at de tror at vi ikke har tenkt til å lage det.

Reply:
:-)  Så spørs det på forventningne for "snart", da.

Reply:
eller "kommer snart". Blir de sittende og vente på at bildet skal komme ? Glad jeg ikke jobber i markedsføring